src/xenia/kernel/xboxkrnl/xboxkrnl_memory.cc

```diff
@@ -425,8 +425,14 @@ DECLARE_XBOXKRNL_EXPORT2(MmQueryAddressProtect, kMemory, kImplemented,
 
 void MmSetAddressProtect_entry(lpvoid_t base_address, dword_t region_size,
                                dword_t protect_bits) {
-  if (!protect_bits) {
-    XELOGE("MmSetAddressProtect: Failed due to incorrect protect_bits");
+  constexpr uint32_t required_protect_bits =
+      X_PAGE_NOACCESS | X_PAGE_READONLY | X_PAGE_READWRITE |
+      X_PAGE_EXECUTE_READ | X_PAGE_EXECUTE_READWRITE;
```

Comment:
I think `X_PAGE_EXECUTE` should be allowed to as it's somewhat like `X_PAGE_NOACCESS` that you can execute, I think, by the way, as long as it's supported in other parts of the emulator's memory management.

Reply:
Added